snafu/benchmarks/uperf/uperf.py

```diff
@@ -312,7 +314,7 @@ def collect(self) -> Iterable[BenchmarkResult]:
         Returns immediately if a sample fails. Will attempt to Uperf run three times for each sample.
         """
 
-        cmd = shlex.split(f"uperf -v -a -R -i 1 -m {self.config.workload}")
+        cmd = shlex.split(f"uperf -v -a -R -i 1 -m {self.config.workload} -P 30000")
```

Comment:
We have to put this PR on hold until we update our deployment bits to also open the port 30000